soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -66,6 +66,19 @@ config SOC_IT8XXX2_CPU_IDLE_GATING
 	  gated by individual drivers. When this option is disabled, CPU idle
 	  mode is always permitted.
 
+config SOC_IT8XXX2_JTAG_ENABLE
+	bool "The JTAG module clock gating is configured here"
```

Comment:
`JTAG module clock gating`, no need to be more verbose here imo

Reply:
Go it, I'll refine it ASAP. Thanks for guidance, sir.